yt/yt/server/node/exec_node/job_proxy_log_manager.cpp

```diff
@@ -3,9 +3,17 @@
 #include "bootstrap.h"
 #include "private.h"
 
+#include <yt/yt/client/api/file_writer.h>
```

Comment:
Includes of `yt/yt/client/...` should be placed between `yt/yt/ytlib/...` and `yt/yt/core/...`